hydra/core/override_parser/overrides_visitor.py

```diff
@@ -125,6 +138,8 @@ def visitPrimitive(
                     ret = False
                 else:
                     assert False
+            elif node.symbol.type == OverrideLexer.ESC:
+                ret = node.symbol.text[1::2]
```

Comment:
do you have a unit test for a single escape token?

Reply:
Yes for instance this one has a single esc token:
```python
pytest.param(r"a\,b", "a,b", id="value:str-esc-comma")
```

Reply:
Ooh right I misunderstood your original question. I actually have the exact same test you suggested which indeed goes through that path (now that I think about it, that's the reason I added it):
```python
 pytest.param(r"\,", ",", id="value:str-esc-comma-alone"),
```